src/apps/competitions/tasks.py

```diff
@@ -494,7 +509,11 @@ def unpack_competition(competition_dataset_pk):
                     continue
                 phase1 = competition['phases'][i - 1]
                 phase2 = competition['phases'][i]
-                if phase2['start'] < phase1['end']:
+                if phase1['end'] is None:
+                    raise CompetitionUnpackingException(
+                        f'Phase: {phase1.get("name", phase2["index"])} must have an end time because it has a phase after it.'
```

Comment:
should be `phase1['index']` not `phase2`

Reply:
great catch